common/config/azure-pipelines/jobs/docs-ci.yaml

```diff
@@ -18,6 +18,11 @@ pr:
     exclude:
       - release/2*
       - release/1*
+  paths:
+    exclude:
```

Comment:
does this make sense? skipping the docs-ci pipeline when there are doc changes??

Reply:
Nope, didn't make sense. I've pushed a change so that the docs CI build will run when there are `.md` or `docs/` changes. I left in the exclusion of the codeowners file though. So if there's only a change to codeowners in a PR, it won't run. (I mostly left this in as a precedent for adding any other exclusions to this config in the future.)